includes/controls/gaps.php

```diff
@@ -60,4 +59,75 @@ protected function get_dimensions() {
 			'row' => esc_html__( 'Row', 'elementor' ),
 		];
 	}
+
+	/**
+	 * Render gaps control output in the editor.
+	 *
+	 * Used to generate the control HTML in the editor using Underscore JS
+	 * template. The variables for the class are available using `data` JS
+	 * object.
+	 *
+	 * @since 3.16.0
+	 * @access public
+	 */
+	public function content_template() {
```

Comment:
Hey @davseve 
 What was the fix? What changed in the `content_template()` that differs from the parent?

Reply:
In the dimensions, we had a condition that tried to print the entire object so I tweaked the condition.

Reply:
@davseve If that is the only change, can you just make a `protected function print_placeholder() {}`
Then in the `gaps.php` file you can just override this small function instead of all this code duplication?